apps/weshipit/fixtures/podcast-episodes.fixture.ts

```diff
@@ -0,0 +1,154 @@
+export interface PodcastEpisode {
+  name: string;
+  title: string;
+  number: number;
+  description: string;
+  slug: string;
+  spotifyLink: string;
+  appleLink: string;
+  companyLogo: string;
+  guestName: string;
+}
+
+export const podcastEpisodes: PodcastEpisode[] = [
```

Comment:
replace this with dynamic content

1. use the rss as a source of truth https://anchor.fm/s/ffc13f2c/podcast/rss
2. transform the RSS to JSON https://github.com/nasa8x/rss-to-json